src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -419,15 +428,25 @@ RoverPositionControl::control_attitude(const vehicle_attitude_s &att, const vehi
 {
 	// quaternion attitude control law, qe is rotation from q to qd
 	const Quatf qe = Quatf(att.q).inversed() * Quatf(att_sp.q_d);
+
+	// Attitude error in Euler angles [rad]
 	const Eulerf euler_sp = qe;
-	// PX4_INFO("Yaw error: %f", double(euler_sp(2)));
+	// PX4_INFO("Yaw error: %f [rad]", double(euler_sp(2)));
 
 	_rates_sp.roll = 0.0;
 	_rates_sp.pitch = 0.0;
-	_rates_sp.yaw = euler_sp(2) / _param_max_turn_angle.get();
+
+	// Apply Attitude P-gain for Yaw control. Euler angle in [rad], rate setpoint in [rad/s]
+	_rates_sp.yaw = euler_sp(2) * _param_att_p.get();
+
+	// PX4_INFO("Yaw rate setpoint in attitude control: %f [rad/s]", (double)_rates_sp.yaw);
```

Comment:
This messed up with me for quite a bit. Apparently the PID tuning page, utilizes the metadata of the parameter as well, so the `GND_ATT_P` parameter having the unit described as `rad` was messing up the slider (it was setting different value than what was shown on the slider), detailed comment here: https://github.com/mavlink/qgroundcontrol/pull/10483/commits/8c85a68bf9e43224d0bf532f821cff39916bcb78#diff-b002776be45737c4f0ac5bde087db4da1b448cabdfa20c6a47eb3e9570db9bdfR43-R45